src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -69,7 +69,14 @@
         programGroup = Metrics.class
 )
 public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
-    static final String USAGE = "Produces from a SAM or BAM a file containing summary alignment metrics";
+    static final String USAGE = "Produces from a SAM or BAM a file containing summary alignment metrics." +
```

Comment:
See line 68.  This USAGE is used in the short usage part of the command line, and so it will go over multiple lines (not good).  You will have to split this into a short and long usage.